kolibri/plugins/management/assets/src/actions.js

```diff
@@ -13,13 +13,61 @@ function setSelectedGroupId({ dispatch }, id) {
   dispatch('SET_SELECTED_GROUP_ID', id);
 }
 
-const classroomCollection = Kolibri.resources.ClassroomResource.getCollection();
-const learnerGroupCollection = Kolibri.resources.LearnerGroupResource.getCollection();
-const learnerCollection = Kolibri.resources.FacilityUserResource.getCollection();
-const memberCollection = Kolibri.resources.MembershipResource.getCollection();
+const ClassroomResource = Kolibri.resources.ClassroomResource;
+const LearnerGroupResource = Kolibri.resources.LearnerGroupResource;
+const MembershipResource = Kolibri.resources.MembershipResource;
+const FacilityUserResource = Kolibri.resources.FacilityUserResource;
+const RoleResource = Kolibri.resources.RoleResource;
+
+function createUser(store, payload, role) {
+  const FacilityUserModel = FacilityUserResource.addModelData(payload);
```

Comment:
We can probably change the name of the method `addModelData` to `createModel` - so that it better describes what it is doing!